### PR TITLE
feat: new homepage

### LINKED ISSUE #353: Update YOUTUBE_ID variable with the correct value

### LINKED ISSUE #377: Redesign Builder homepage
We drop about 22% of users at the homepage. For first-time users, the homepage fails in a few ways:
1. It doesn't show the experience of the Builder or the output of the Builder (what can I do here? What will I gain?)
2. It shows a very 'cold' state when there are no projects and buries the most important row of buttons—the `TEMPLATE` row—at the bottom of the stack.

Design: 
1 home empty copy 3

Copy: https://docs.google.com/document/d/1uPAvyZYkyE52O32XC17k5i9sqgmFZfDjaetH5sidzdw/edit?usp=sharing

Translations: [CN and KR here](https://docs.google.com/document/d/1R3FFztIWLDfolX2eo8k1NHXZZMQK36WSwy50GBnQMkk/edit?usp=sharing)

### PR DESCRIPTION
**Pull request by belohlavek**
Kapture 2019-03-25 at 18 19 44

Closes #377  
Closes #353

### CODE REVIEW

File: src/components/ProjectCard/ProjectCard.tsx [hunk 1]
```diff
@@ -84,7 +85,7 @@ export default class ProjectCard extends React.PureComponent<Props, State> {
         <div className="project-data">
           <div className="title">{project.title}</div>
           <div className="description" title={project.description}>
-            {getProjectDimensions(project)}
+            {getProjectDimensions(project)} {items > 0 && `- ${items} items`}
```
Comment (cazala) [inline src/components/ProjectCard/ProjectCard.tsx]: i18n

File: src/components/App/App.tsx [hunk 2]
```diff
@@ -1,16 +1,23 @@
...
 export default class App extends React.PureComponent<Props> {
   render() {
-    const { children } = this.props
+    const { isHomePage, children } = this.props
+
     return (
-      <DappApp activePage="builder" locales={Object.keys(languages)} onSignIn={undefined}>
-        {children}
-      </DappApp>
+      <>
```
Comment (cazala) [inline src/components/App/App.tsx]: Should we add `isFullscreen` and `isOverlay` to `<App>` and use that one instead?

File: src/components/HomePageHero/HomePageHero.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  }
+
+  handleWatchVideo = () => {
+    this.props.onOpenModal('VideoModal')
+  }
+
+  render() {
+    return (
+      <Hero height={472} className="HomePageHero">
+        <div className="container">
```
Comment (cazala) [inline src/components/HomePageHero/HomePageHero.tsx]: This could be part of `<Hero>`

File: src/components/ProjectCard/ProjectCard.css [hunk 4]
```diff
@@ -1,23 +1,34 @@
 .ProjectCard {
   position: relative;
+  width: var(--homepage-card-width);
+  height: var(--homepage-card-height);
+  border-radius: 12px;
+  margin-top: var(--homepage-card-margin);
+  margin-right: var(--homepage-card-margin);
+  background-color: rgba(var(--purplish-grey-raw), 0.16);
```
Comment (cazala) [inline src/components/ProjectCard/ProjectCard.css]: This var has been renamed to `bluish-steel-raw`

File: src/routing/Routes.tsx [hunk 5]
```diff
@@ -3,16 +3,16 @@ import { Switch, Route, Redirect } from 'react-router-dom'
 import { Responsive } from 'decentraland-ui'
 import { env } from 'decentraland-commons'
 import Intercom from 'decentraland-dapps/dist/components/Intercom'
+import { default as DappApp } from 'decentraland-dapps/dist/containers/App'
```
Comment (cazala) [inline src/routing/Routes.tsx]: Couldn't this just be `App` instead of `{ default as DappApp }`?

Comment (belohlavek) [reply src/routing/Routes.tsx]: yus

**Pull request merged**